plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -77,6 +77,16 @@ typedef void (*proxy_function_prototype)(void* userData, AL::usdmaya::nodes::Pro
 
 const char* ProxyShape::s_selectionMaskName = "al_ProxyShape";
 
+//----------------------------------------------------------------------------------------------------------------------
+MDagPath ProxyShape::parentTransform()
```

Comment:
You sure you really need this?  You're inheriting from MayaUsdProxyShapeBase, which has the exact same implementation of parentTransform():

https://github.com/Autodesk/maya-usd/blob/dev/lib/nodes/proxyShapeBase.cpp#L921